web/apps/admin/src/components/Sidebar/sidebar.module.css

```
@@ -6,6 +6,10 @@
   width: 220px !important;
 }
 
+.sidebar :global(.sidebar-module_main__qDvo4) {
```

Stylelint (17.11.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)